src/DRICH_geo.cpp

```diff
@@ -204,6 +222,14 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
     vesselSolid = vesselBox;
     gasvolSolid = gasvolBox;
     break;
+  case 4:
+    vesselSolid = vesselBox;
+    gasvolSolid = gasvolBox;
+    break;
+  case 5:
+    vesselSolid = vesselBox;
+    gasvolSolid = gasvolBox;
+    break;
```

Comment:
Identical case blocks can be combined for readability:
```suggestion
  case 4:
  case 5:
    vesselSolid = vesselBox;
    gasvolSolid = gasvolBox;
    break;
```

Reply:
This may be applied beyond the context range where github allows suggested changes.